doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -2597,138 +2601,150 @@ PSA Crypto support
              - Supported
              - Supported
-           * - **SSF crypto service**
-             - --
-             - --
-             - --
-             - --
-             - --
-             - --
+
       .. tab:: nRF53 Series
 
         .. list-table:: Cryptographic support
            :header-rows: 1
            :widths: auto
 
-           * - Feature
+           * - Cryptographic implementation
+             - Driver
              - nRF5340
-           * - **Oberon PSA core - nrf_cc3xx driver**
+           * - :ref:`Oberon PSA Crypto <ug_crypto_architecture_implementation_standards_oberon>`
+             - :ref:`nrf_cc3xx <crypto_drivers>`
              - Supported
-           * - **Oberon PSA core - cracen driver**
+           * - :ref:`Oberon PSA Crypto <ug_crypto_architecture_implementation_standards_oberon>`
+             - :ref:`CRACEN <crypto_drivers>`
              - --
-           * - **Oberon PSA core - nrf_oberon driver**
+           * - :ref:`Oberon PSA Crypto <ug_crypto_architecture_implementation_standards_oberon>`
+             - :ref:`nrf_oberon <crypto_drivers>`
              - Supported
-           * - **TF-M crypto service**
+           * - :ref:`TF-M Crypto Service <ug_crypto_architecture_implementation_standards_tfm>`
+             - :ref:`Oberon PSA Crypto drivers <crypto_drivers>`
              - Supported
-           * - **SSF crypto service**
-             - --
+
       .. tab:: nRF54 Series
 
         .. list-table:: Cryptographic support
            :header-rows: 1
            :widths: auto
 
-           * - Feature
+           * - Cryptographic implementation
+             - Driver
              - nRF54H20
              - nRF54L05
              - nRF54L10
              - nRF54L15
-           * - **Oberon PSA core - nrf_cc3xx driver**
+           * - :ref:`Oberon PSA Crypto <ug_crypto_architecture_implementation_standards_oberon>`
+             - :ref:`nrf_cc3xx <crypto_drivers>`
              - --
              - --
              - --
              - --
-           * - **Oberon PSA core - cracen driver**
+           * - :ref:`Oberon PSA Crypto <ug_crypto_architecture_implementation_standards_oberon>`
+             - :ref:`CRACEN <crypto_drivers>`
              - --
              - Supported
              - Supported
              - Supported
-           * - **Oberon PSA core - nrf_oberon driver**
+           * - :ref:`Oberon PSA Crypto <ug_crypto_architecture_implementation_standards_oberon>`
+             - :ref:`nrf_oberon <crypto_drivers>`
              - --
              - Supported
              - Supported
              - Supported
-           * - **TF-M crypto service**
+           * - :ref:`TF-M Crypto Service <ug_crypto_architecture_implementation_standards_tfm>`
+             - :ref:`Oberon PSA Crypto drivers <crypto_drivers>`
              - --
              - --
              - --
              - Experimental
-           * - **SSF crypto service**
+           * - SSF Crypto Service
```

Comment:
I suggest we remove the line about SSF crypto service.
This can be reintroduced and renamed later